pb_model/fields.py

```diff
@@ -103,6 +104,32 @@ def from_pb(instance, dj_field_name, pb_field, pb_value):
         raise NotImplementedError()
 
 
+class ProtoBufAnyField(models.BinaryField, ProtoBufFieldMixin):
```

Comment:
do we need to swap the sequence of inheritance to prevent `models.BinaryField` overwritten `ProtoBufFieldMixin` ?